api/client.go

```diff
@@ -0,0 +1,56 @@
+package api
```

Comment:
the `/api` folder is usually used for OPENAPI (Swagger) docs.  This looks like it might belong nested under `/internal`

Reply:
Not really. This is a normal pattern to interact with an API. internal should only be stuff thats related to the binary itself.  It's basically following this pattern https://github.com/cli/cli

Reply:
I was going off of this standard: https://github.com/golang-standards/project-layout/tree/master/api

But it's not a point worth worrying about.

Reply:
Oh nice. I didn't even know that existed. Good call out though. I think it still fits that description though. This is how i've seen api used in most cases.